examples/graphql-context/graphql_context.bal

```diff
@@ -3,18 +3,17 @@ import ballerina/http;
 import ballerina/lang.value;
 
 @graphql:ServiceConfig {
-    // Initialization of the `graphqlContext` should be provided to the
-    // `contextInit` field.
+    // Initialization of the `graphqlContext` should be provided to the `contextInit` field.
     contextInit: isolated function (http:RequestContext requestContext,
                                     http:Request request)
                                     returns graphql:Context|error {
```

Comment:
We can re-format this to have 100 characters max in the line. We have splitter this to match the previous 80 characters-per-line rule.

Reply:
Yes, the character limit of the new BBE design is 100.